traildb/traildb.py

```diff
@@ -414,6 +415,42 @@ def create_filter(self, event_filter):
         return TrailDBEventFilter(self, event_filter)
 
 class TrailDBEventFilter(object):
+    """
+    Converts a query defined in terms of Python collections to a
+    `tdb_event_filter` which can be passed to various TrailDB functions.
+    Performs some validation when parsing the query.
+
+    Queries are boolean expressions defined from terms and clauses.  A term is
+    defined using a tuple:
+
+    (field_name, "value") -- match records with field_name == "value"
+    (field_name, "value", False) -- match records with field_name == "value"
+    (field_name, "value", True) -- match records with field_name != "value"
```

Comment:
If it supported 2-tuples it would still be backwards compatible, no? Could also define this as a constant for this like `(field_name, "value", traildb.EQUAL)`